service/service.go

```diff
@@ -874,17 +875,16 @@ func setLogFields(ctx context.Context, fields log.Fields) context.Context {
 }
 
 func getLogFields(ctx context.Context) log.Fields {
-	if ctx == nil {
```

Comment:
Do we still need this nil check? Any callers of this function ever passing a nil context which will cause nil pointer dereference error when we call `ctx.Value` below.